app/components/mandatarissen/update-state.js

```diff
@@ -162,14 +163,14 @@ export default class MandatarissenUpdateState extends Component {
     const endDate = this.args.mandataris.einde;
 
     const newMandatarisProps = {
-      bekleedt: this.args.mandataris.bekleedt,
-      beleidsdomein: this.selectedBeleidsdomeinen,
-      isBestuurlijkeAliasVan: this.args.mandataris.isBestuurlijkeAliasVan,
       rangorde: this.rangorde,
-      isDraft: true,
       start: this.date,
-      status: this.newStatus,
       einde: endDate,
+      bekleedt: this.args.mandataris.bekleedt,
+      isBestuurlijkeAliasVan: this.args.mandataris.isBestuurlijkeAliasVan,
+      beleidsdomein: this.selectedBeleidsdomeinen,
+      status: this.newStatus,
+      publicationStatus: await getDraftStatus(this.store),
```

Comment:
I think we need to check this business rule, because it seems weird to me that if you do an update state, your new mandataris becomes draft, this doesn't really seems to make sense, because draft is meant for input of an installatievergadering, but I suppose if you do update state the installatievergadering already passed.